src/network/networkd-link-bus.c

```diff
@@ -10,11 +10,42 @@
 static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_operational_state, link_operstate, LinkOperationalState);
 static BUS_DEFINE_PROPERTY_GET_ENUM(property_get_administrative_state, link_state, LinkState);
 
+static int property_get_bit_rates(
+                sd_bus *bus,
+                const char *path,
+                const char *interface,
+                const char *property,
+                sd_bus_message *reply,
+                void *userdata,
+                sd_bus_error *error) {
+
+        Link *link = userdata;
+        Manager *manager;
+
+        assert(bus);
+        assert(reply);
+        assert(userdata);
+
+        manager = link->manager;
+
+        if (!manager->use_speed_meter ||          /* disabled */
+            manager->speed_meter_usec_old == 0 || /* not started yet */
+            !link->stats_updated)                 /* error in getting link stats */
+                return sd_bus_message_append(reply, "(dd)", -1, -1);
```

Comment:
Shouldn't this be -1., -1. ?

Reply:
What do you mean? Should we return some error, e.g. `-EOPNOTSUPP`?